src/routes/JobOpportunity/Competences.tsx

```diff
@@ -4,12 +4,18 @@ import { useActionBar } from '@/hooks/useActionBar';
 import { Accordion, Button } from '@jod/design-system';
 import { createPortal } from 'react-dom';
 import { useTranslation } from 'react-i18next';
+import { useRouteLoaderData } from 'react-router-dom';
 import Tabs from './Tabs';
-import { mockTyomahdollisuus } from './utils';
+import { LoaderData } from './loader';
 
 const Competences = () => {
   const { i18n, t } = useTranslation();
-  const title = mockTyomahdollisuus.nimi;
+  const data = useRouteLoaderData('job-opportunity') as LoaderData;
+  const lang = i18n.language;
+  const title = data.tyomahdollisuus?.otsikko[lang] ?? '';
+  const competences = data.osaamiset;
+  console.log(competences);
```

Comment:
Console logs should be removed

Reply:
Good catch; fixed, also added a linter rule.